Behavior/Resources/Localizable.xcstrings

```diff
@@ -349,6 +352,7 @@
     },
     "NOTIFICATION_PERMISSIONS_TITLE" : {
       "comment" : "MARK: Notifications",
+      "extractionState" : "stale",
```

Comment:
Please remove all stale localization values.

If you want to learn more about localization, please check out this video for more details: https://developer.apple.com/wwdc23/10155.